docs/integrations/optuna.rst

```diff
@@ -3,150 +3,216 @@
+
+Step 2: Create an Experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below to create a Neptune experiment:
 
 .. code-block:: python3
 
-    import neptunecontrib.monitoring.optuna as opt_utils
-
-    neptune_callback = opt_utils.NeptuneCallback()
-
-Pass **neptune_monitor** to **study.optimize**
-----------------------------------------------
-Monitor the metrics and parameters checked at each run:
-
-.. code-block:: python3
+    neptune.create_experiment('optuna-sweep')
 
-    study = optuna.create_study(direction='maximize')
-    study.optimize(objective, n_trials=100, callbacks=[neptune_callback])
+Open the link in a new tab. The charts will currently be empty, but keep the window open. You will be able to see live metrics once logging starts.
 
-Log all results
----------------
-You can log additional information from optuna study after the sweep has completed.
-By running:
+Step 3: Create the Neptune Callback
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note:: You can pass the experiment object as the first argument if you want to do it explicitly.
 
 .. code-block:: python3
 
-    opt_utils.log_study_info(study)
+   import neptunecontrib.monitoring.optuna as opt_utils
 
-You log the following things to Neptune:
+   neptune_callback = opt_utils.NeptuneCallback()
 
-* Best score
-* Best parameters
-* Interactive plotly figures from optuna.visualization: plot_contour, plot_slice, plot_parallel_coordinate, optimization_history
-* Pickled study object
-
-.. image:: ../_static/images/integrations/optuna_charts.gif
-   :target: ../_static/images/integrations/optuna_charts.gif
-   :alt: Optuna charts in Neptune
-
-Log study and charts after every iteration
-------------------------------------------
-You can log interactive charts from optuna.visualization and the study object after every iteration.
-Simply specify what you want to log at `NeptuneCallback` creation.
+Step 4: Pass the Neptune Callback as a callback to ``study.optimize``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Monitor the metrics and parameters checked at each run:
 
 .. code-block:: python3
 
-    neptune_callback = opt_utils.NeptuneCallback(log_study=True, log_charts=True)
+   study = optuna.create_study(direction='maximize')
+   study.optimize(objective, n_trials=100, callbacks=[neptune_callback])
 
-
-Monitor your Optuna training in Neptune
----------------------------------------
-Now you can watch your Optuna hyperparameter optimization in Neptune!
+Step 5: Monitor your Optuna training in Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now you can switch to the Neptune tab which you had opened previously to watch the optimization live!
 
 Check out this |example experiment|.
 
 .. image:: ../_static/images/integrations/optuna_monitoring.gif
    :target: ../_static/images/integrations/optuna_monitoring.gif
    :alt: Optuna monitoring in Neptune
-
-Full script
------------
+   
+Advanced Logging
```

Comment:
Please change to Advanced options as that is the naming we'd like to use across those integrations